UnitOfWork/Models/BaseEntity.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace UnitOfWork.Models;
+
+public class BaseEntity
+{
+    public int Id { get; set; }
+    public DateTime CreatedAt { get; set; }
+    public DateTime UpdatedAt { get; set; }
```

Comment:
Make the field nullable: `public DateTime? UpdatedAt { get; set; } = default;`

Reply:
fixed